resources.md

```diff
@@ -125,6 +125,13 @@ to get started.
 
 <div markdown="1" class="item">
 
+#### [Datadog integration](https://docs.datadoghq.com/integrations/trino/)
+
+The Datadog integration allows the observability service for cloud-scale
+applications to monitor your Trino cluster. It accesses the JMX metrics provided
```

Comment:
Can we include link to JMX metrics documentation ?

Reply:
Done